src/xinterpreter.cpp

```diff
@@ -65,12 +66,16 @@ interpreter::interpreter(int argc, char* argv[])
     // When building with Emscripten, pass --no-readline to disable
     // readline support, as r-base is not compiled with readline
     // and will not read input from the command line.
-    #ifdef __EMSCRIPTEN__
-        const char* argvNew[] = {"--no-readline"};
-        Rf_initEmbeddedR(sizeof(argvNew) / sizeof(argvNew[0]), const_cast<char**>(argvNew));
-    #else
-        Rf_initEmbeddedR(argc, argv);
-    #endif
+#ifdef __EMSCRIPTEN__
+    char *argvNew[] = {
```

Comment:
This should be `const char*`, and the cast below can be removed (I think the (char*) cast should be removable too).